cmd/carbonapi/http/render_handler.go

```diff
@@ -260,6 +261,9 @@ func renderHandler(w http.ResponseWriter, r *http.Request) {
 		// Otherwise it should be 500
 		errMsgs := make([]string, 0)
 		for _, err := range errors {
+			if err != ztypes.ErrNoMetricsFetched {
```

Comment:
It's better to use `!merry.Is(err, ztypes.ErrNoMetricsFetched)` here as `!=` will only compare one level of the errors, but "Is" will check if on any level there is an error of that type. That's not too important, but would allow to change all types of the errors much easier in future.

Also I'm a bit confused with the PR's name - you are saying that you want to ignore "no metircs error", but here you are ignoring everything, but "NoMetircsFetched".

Reply:
Yes, it should be `merry.Is(err, ztypes.ErrNoMetricsFetched)`

Reply:
![image](https://user-images.githubusercontent.com/3659110/78258888-7891fc00-752e-11ea-9e79-e8e2875c33b8.png)

But it looks like this error is not equal to ErrNoMetricsFetched, it is equal to https://github.com/go-graphite/carbonapi/blob/c538fcd86be320b360e62fd0a6dd7d136b428e87/expr/expr.go#L129

We may need to modify the error handling here.